.env.example

```diff
@@ -0,0 +1,3 @@
+TENANT="your-tenant-value"
+OAUTH_KEY="your-oauth-key"
+OAUTH_SECRET="your-oauth-secret"
```

Comment:
If I understand correctly, this file will be used as the basis for the user's .env file, which in turn will provide environment variables to Docker?

We should avoid having secrets in plaintext (even on the user's machine).  And I get that this is sample code, but it would be nice if we could reinforce best practices...

One approach would be to encrypt the secret (e.g., using openssl) and store it base-64 encoded in the .env file. Within the Dockerfile, you'd use openssl to decrypt the environment variable.  It's not perfect -- e.g., you'd likely have to pass the decryption key via an env var.  But if it were defined in your own environment (and not committed to a file), it would definitely lower the exposure.

Let me know if this is something you want to do in this work item, as a tech debt card to follow, etc.

Reply:
one never wants to decline a security consideration, I suppose we'd want to consider the cost to ease-of-use for the target audience in this case client developers.  putting secrets in .env for local dev is a common strategy that we're using in multiple projects, I'd suggest putting it up to the team for discussion.

Reply:
Sure, that would work.  If it's purely for local dev, it's not a huge deal.  The problem is that local-dev practices often find their way into prod. :)